config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/bundle/loader/ServiceAndPolicyLoaderUtil.java

```diff
@@ -53,8 +55,9 @@ public static String handleDuplicatePathName(Map<String, ? extends Folderable> b
         int duplicateCounter = 2;
         String basePath = entity.getPath();
         String clonePath = basePath;
+        String sysProp = System.getProperty(HANDLE_DUPLICATE_NAMES);
 
-        if (System.getProperty("exportDuplicate").equals("true")) {
+        if (sysProp != null && sysProp.equals("true")) {
```

Comment:
use Boolean.getBoolean, @J-Lou  it has the same effect with less code =)

Reply:
Neat!